### ISSUE TITLE
support editors other than VSCode

### ISSUE THREAD

**Issue by eyy** [forge42dev/Remix-Dev-Tools#144]
it would be nice if the `open in VSCode` buttons would be more generic, and could open the file/dir in other editors.
Our team, e.g., is using GoLand.

it's not a big change; i changed `openInVsCode()`, in `src/vite/plugin.tsx`, to the following, and it works:
```js
const openInVsCode = (path2, lineNum) => {
  const p = normalizePath(process.cwd() + '/' + path2);
  exec(`goland "${p}" --line ${lineNum}`);
};
```
but surly it needs to be configurable; maybe pass `remixDevTools` an option like `openEditor(path: string, line: number): string`, and `exec`ute the return value. but i'm not sure what's the best way.

anyway, thanks for this cool project!

**Comment by andrewbellucci** [comment]
Just found out that this tool exists **today** and I've been using Remix for quite some time! I was actually about to make an issue for this, because I use WebStorm and GoLand as well. It would be extremely nice to have this. I'm going to draft a PR today to allow for this. Thanks for the investigation @eyy

**Comment by AlemTuzlak** [comment]
this has been merged and is available from 4.6.2

**Comment by eyy** [comment]
@andrewbellucci @AlemTuzlak thanks a lot!
